.github/workflows/scripts.yml

```diff
@@ -0,0 +1,40 @@
+name: CI with script
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install packages
+        run: sudo apt install -y --no-install-recommends pkg-config curl
+
+      - name: Install sail from binary
+        run: |
+          sudo mkdir -p /usr/local
+          curl --location https://github.com/rems-project/sail/releases/download/0.19-linux-binary/sail.tar.gz | sudo tar xvz --directory=/usr/local --strip-components=1
+
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: Build simulator
+        run: ./build_simulators.sh
+        # Build with ./build_emulators.sh
+
+      - name: Test simulator
+        run: ctest --test-dir build --output-junit tests.xml --output-on-failure
+
+      - name: Upload test results
```

Comment:
I think we don't need to upload the test results/event payload since the other workflow will already do this.